README.md

```diff
@@ -1 +1,139 @@
 # RepreZen Swagger Parser
+
+## Overview ##
+
+The RepreZen Swagger Parser is a Java-based validating parser for
+OpenAPI 3.0 offering full compliance with the
+[OpenAPI 3.0 Specification](https://github.com/OAI/OpenAPI-Specification/blob/3.0.0-rc0/versions/3.0.md),
+and a highly uniform read/write programming API.
+
+Feature highlights include:
+* **High Performance** - Informal testing shows a 3x-4x performance
+  improvement over the current Java Swagger 2.0 parser. This is
+  largely attributable to a design based on adapting Jackson
+  `JsonNode` objects, rather than deserializing to internal POJOs.
+  
+* **Read/Write API** - All aspects of a model may be interrogated and
+  modified through the API. We also provide fluent builders for all
+  API object types. API provides bidirectional navigation through the
+  model, and every object that is a property value of its containing
+  object (whether as a named field or a map entry) knows its own key.
+  
+* **Tolerant Reader** - The parser yields a fully accessible result
+  from any valid JSON or YAML input - whether or not the input is a
+  valid OpenAPI specification.
+  
+* **Separate, extensible validation** - All validation beyond base
+  JSON/YAML parsing is performed after the initial parse, and it can
+  be disabled for speed. API-level validation is separated from
+  injectable implementation-level validation. The latter, for example,
+  is where unrecognized JSON properties can be noted, as these are not
```

Comment:
Confused here about "implementation-level validation".  Implementation of... the parser? Also confused about "unrecognized JSON properties can be noted." Noted as errors? Or noted, as in, you can see the properties and their values?

My best guess is that this category of validation checks the input spec for rules that are stated in the OpenAPI v3 spec, but are not expressible as structural validation rules.  If this is correct, maybe we could call this "logical validation" or "rule-based validation".